backend/controllers/AssetsController.ts

```diff
@@ -2,20 +2,35 @@ import { Express, Request, Response } from "express";
 import { BaseController } from "./BaseController";
 import * as AssetsHandler from "../handlers/AssetsHandler";
 import { HttpStatusCode } from "axios";
+import multer from "multer";
 import { HttpError } from "../types/HttpError";
 
 export class AssestsController extends BaseController {
+  private upload: multer.Multer;
+
   constructor(app: Express) {
     super(app, "/v2/assets");
   }
 
   protected initializeRoutes(): void {
-    this.router.post("/", this.postAssets.bind(this));
+    this.upload = multer({ storage: multer.memoryStorage() });
```

Comment:
This is fine, but it would be nice if we could just stream it through.

Reply:
Agreed. I need to read up on exactly how to do that; for now, this gets things working with the refactor.